_pages/dev/features/scroll-to-top.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Scroll to Top
+feature:
+  - name: Scroll to Top
+    spa_version: 5.0
+    cx_version: 2105
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 5.0 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The scroll to top feature allows users to quickly scroll back to the top of the page.
+
+---
+
+**Table of Contents**
+
+- This will become a table of contents (this text will be scrapped).
+{:toc}
+
+---
+
+## Enabling Scroll to Top
+
+Scroll to top is part of the `@spartacus/storefront` library, and installed during the initial setup. To enable this feature in your storefront add the scroll to top CMS data to a content slot using ImpEx. 
+
+### CMS Component
+
+Scroll to top is CMS-driven and consists of the following CMS component:
+
+- `ScrollToTopComponent`
+
+If you are using the version 5.0 or greater of [{% assign linkedpage = site.pages | where: "name", "spartacussampledata-extension.md" %}{{ linkedpage[0].title }}]({{ site.baseurl }}{% link _pages/install/spartacussampledata-extension.md %}), the scroll to top component is already enabled. However, if you decide not to use the `spartacussampledata` extension, you can enable scroll to top through ImpEx.
```

Comment:
if you decide not to use the `spartacussampledata` extension, you can create the Scroll to Top CMS component with ImpEx.